.pre-commit-config.yaml

```diff
@@ -36,10 +36,34 @@ repos:
         # document contents. For example
         # packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/ruby/changeCondition.yml
         exclude: ^packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/.*/[^/]*\.yml$
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: "v2.7.1"
+  - repo: local
     hooks:
       - id: prettier
+        name: prettier
+        types: [text]
+        language: node
+        entry: prettier --write --list-different --ignore-unknown
+        additional_dependencies:
+          - prettier@2.7.1
```

Comment:
Why not use the technique here? https://github.com/pre-commit/mirrors-prettier/issues/29#issuecomment-1332667344